File: DAY 12/overLoading.py
class calculator:
    def add(self,a,b,c=None):
        if c is not None:
            return a+b+c
        return a+b
c=calculator()
print(f"{c.add(1,3)},{c.add(1,3,6)},{c.add(4.2,3.6)}")

File: DAY 12/overLoading.java
public class overLoading{
    static class calculator{
        int add(int a,int b){
            return a+b;        
        }
        int add(int a,int b,int c){
            return a+b+c;        
        }
        double add(double a,double b){
            return a+b;        
        }
    }
    public static void main(String[] args){
        calculator c=new calculator();
        System.out.println(c.add(1,2)+"  "+c.add(4,6,2)+"  "+c.add(3.24,7.54));
    }
}